js/modal.js

```diff
@@ -259,11 +260,12 @@
 
   Modal.prototype.setScrollbar = function () {
     var bodyPad = parseInt((this.$body.css('padding-right') || 0), 10)
+    this.originalBodyPad = this.$body.attr('style') ? bodyPad : null
```

Comment:
Yup, that's why I haven't merged it yet. Was thinking about checking it with `/padding-right/i`, what do you think?